sdfObject/sdfobject-genericdefaulttransitiontime.sdf.json

```diff
@@ -1,7 +1,7 @@
 {
   "info": {
     "title": "Example Bluetooth mesh Generic Default Transition Time Model", 
-    "version": "20200710", 
+    "version": "20210-03-05", 
```

Comment:
While version strings are not well-defined in SDF 1.1, this looks like a typo.

Reply:
fixed